doc/library/config.txt

```diff
@@ -728,6 +728,23 @@ import theano and print the config variable, as in:
 
     Location of the magma library.
 
+.. attribute:: config.ctc.enabled
+
+    Default: ``'False'``
+
+    If ``'True'``, use `warp-ctc <https://github.com/baidu-research/warp-ctc>`_ for CTC
+    loss computations.
+
+    If ``'False'``, disable warp-ctc.
```

Comment:
This configuration value makes no sense to me.  Either you use the CTC ops in your graph and you always want them or you don't and you don't want them.  There will never be a case where you use the ops, but want to disable them.